t/SD_Protocols/02_mcBit2Hideki.t

```diff
@@ -59,6 +59,17 @@ subtest 'message is to long' => sub {
 	my $bitdata='10101000110000100011001110110101001110100011111000001010000001111000001100000000000000000000000000000000000000000000000000001111111111111111111111111111111111111111111111111111111111000000000000000000000000000000000000101010100000000000000000000000000';
 
 	($rcode,$hexresult)=$Protocols->mcBit2Hideki(undef,$bitdata,$id,length $bitdata);
-	is($rcode,-1,'check returncode for single OSV1 transmission');
+	is($rcode,-1,'check returncode for Hideki transmission');
 	is($hexresult,' message is to long','check result message to long');
-};
\ No newline at end of file
+};
+
+
+subtest 'message is inverted' => sub {
+	plan(2);
+
+	my $bitdata='10101000111001001111010001011010111011011111010000001010111110010111010110010101011101100110';
+
+	($rcode,$hexresult)=$Protocols->mcBit2Hideki(undef,$bitdata,$id,length $bitdata);
+	is($rcode,1,'check returncode for OSV1 transmission');
```

Comment:
@sidey79 hier müsste noch der Name angepasst werden.

Es müsste korrekter Weise
`is($rcode,1,'check returncode for Hideki transmission');`
lauten.

Reply:
erledigt